### ISSUE TITLE
Error Handling if Enter is pressed when prompted for Hostname

### ISSUE THREAD

**Issue by cruisemaniac** [runeasymail/easymail#155]
The installer proceeds with `NULL` value for hostname if user presses enter (without entering any value) when asked to "Enter Hostname".

System should not proceed or should Ideally pick up the -H parameter that is provided when docker container is created:

```
docker run -it -p=110:110 -p=25:25 -p=995:995 -p=8080:80 -p=44380:443  -p=587:587 -p=993:993 -p=143:143 -p=465:465 -h "mail.example.com" --name="easymail" -v /etc/ssl/certs/:/etc/ssl/certs/ ubuntu:16.04 /bin/sh -c "if [ -f /run.sh ]; then bash /run.sh; fi; exec /bin/bash"
```
What / where is the -h paramter used for ?

**Comment by pavel-tashev** [comment]
Can you propose a solution?

**Comment by cruisemaniac** [comment]
lets start with a `set -e`

and simple traps...
```
if [[ -n "$input" ]] ; then
  echo "error happened with input"
  exit 1
fi
```
Let me know if this will work?

**Comment by cruisemaniac** [comment]
And I'd presume, that since we're heading with docker, all inputs should be captured BEFORE SCRIPT START...

**Comment by GyunerZeki** [comment]
This should be applicable to post_install.sh too.

The current command for running is
```
docker run -itd  -p=110:110 -p=25:25 -p=995:995 -p=80:80 -p=443:443 -p=587:587 -p=993:993 -p=143:143 -p=465:465 -h "test.example.com" --name="easymail" XXXXXX /bin/sh -c "bash /opt/easymail/post_install.sh test.example.com; exec /bin/bash"
```
We can make that to 
```
docker run -itd  -p=110:110 -p=25:25 -p=995:995 -p=80:80 -p=443:443 -p=587:587 -p=993:993 -p=143:143 -p=465:465 -h "test.example.com" --name="easymail" XXXXXX /bin/sh -c "bash /opt/easymail/post_install.sh; exec /bin/bash"
```
And after that inside in post_install.sh we can take the "test.example.com" from the hostname.

Is that your proposal?

**Comment by cruisemaniac** [comment]
yes that is...

**Comment by GyunerZeki** [comment]
That's great. I totally agree with you. Can you provide a Pull request for this changes?

**Comment by GyunerZeki** [comment]
@cruisemaniac the issue is resolved on v1.0.8, closing the ticket.
